apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryBigNumeric.cs

```diff
@@ -51,40 +42,38 @@ public struct BigQueryBigNumeric : IEquatable<BigQueryBigNumeric>, IComparable<B
         /// <summary>
         /// The maximum valid value for <see cref="BigQueryBigNumeric"/>, equal to (2^255 - 1)e-38.
         /// </summary>
-        public static BigQueryBigNumeric MaxValue { get; } = new BigQueryBigNumeric(s_maxValue);
+        public static BigQueryBigNumeric MaxValue { get; } = new BigQueryBigNumeric(s_numericType.MaxValue);
 
         /// <summary>
         /// The minimum valid value for <see cref="BigQueryBigNumeric"/>, equal to -2^255e-38.
         /// </summary>
-        public static BigQueryBigNumeric MinValue { get; } = new BigQueryBigNumeric(s_minValue);
+        public static BigQueryBigNumeric MinValue { get; } = new BigQueryBigNumeric(s_numericType.MinValue);
 
         /// <summary>
         /// The smallest positive value for <see cref="BigQueryBigNumeric"/>, equal to 1e-38.
         /// </summary>
-        public static BigQueryBigNumeric Epsilon { get; } = new BigQueryBigNumeric(1);
+        public static BigQueryBigNumeric Epsilon { get; } = new BigQueryBigNumeric(s_numericType.Epsilon);
 
-        // Integer representation, always scaled by 38 decimal places - so a value of 1 here represents
-        // a numeric value of 1e-38 for example.
-        private readonly BigInteger _integer;
+        // General purpose numeric representation. Should always be of type s_numericType unless
```

Comment:
Not sure I understand about the default value.

Reply:
Ah, I now get it - in the default value of *this* type, the field value will be the default value of `NumericValue`.